project_setup/README.md

```diff
@@ -110,50 +110,35 @@ usage of the tool is:
 This file will now contain definitions for all the Ansible roles.  Edit
 the file, and remove any role that will not be required for your project.
 
-## Terraform IAM Credentials to Travis Tool 🔑‍👉👷🏻 ##
+## Terraform IAM Credentials to GitHub Secrets 🔑‍👉🤫 ##
 
-When Travis-CI needs credentials to run we provide them in its `.travis.yml`
-file in an encrypted format.  Extracting fresh IAM credentials from a
-Terraform run,
-[encrypting them properly](https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml)
-, and then formatting them into well-formed `yml` that will make the linters
-happy is no small task.
+When GitHub Actions workflows require credentials to run we provide them via
+[secrets](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets).
+This usually involves extracting the secrets from the Terraform state
+json output.  Then some pointing, clicking, cutting and pasting on the
+repository's settings.
 
-To simplify this task use the [`iam-to-travis`](scripts/iam-to-travis) tool
-located in the [`scripts`](scripts) directory.  The tool will output `yml`
-that can be pasted directly into a `.travis.yml` file.  The tool has
-adjustable indentation and width with reasonable defaults.
+To simplify this task use the [`iam-to-github`](scripts/iam-to-github) tool
+located in the [`scripts`](scripts) directory.  The tool will create secrets
+using your
+[personal access token (PAT)](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line).
+Note: Your PAT requires the "repo" scope to be set.
 
 Execute the tool from your GitHub project's terraform directory:
 
 ```bash
-iam-to-travis
+iam-to-github xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
What are all the `x` characters denoting?

Reply:
https://github.com/cisagov/development-guide/blob/bc8dfe5ddfff2a6ec75d8cc7458499d3378e609d/project_setup/scripts/iam-to-github#L12

Reply:
I made this clearer in bac7c9d.